README.md

```diff
@@ -635,6 +635,22 @@ The `classes` key as a root key of the parameter definitions takes on special me
 Example class as value:
 ```yml
 classes:
+  animals:
+    type: definition
+    description: Animals included in model
+    fields:
+      goes:
+        type: any
+        description: What noise does the animal make?
+        default: oink
+      is_mammal:
+        type: boolean
+        description: Is this animal a mammal
+        default: false
+  default:
```

Comment:
the `default` section looks off an indent level to me

Reply:
should sheep be in our default animals?

Reply:
Yeah, I guess just making the animals used all defaults and not re-define any animals would work the best probably